packages/terra-status-view/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+* Changed
+  * Removed focus from glyphs in IE10 .
+
```

Comment:
Should be moved under `Unreleased` section

Reply:
Done 322f801